doc/classes/OpenXRAPIExtension.xml

```diff
@@ -0,0 +1,109 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="OpenXRAPIExtension" inherits="RefCounted" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		Makes the OpenXR API available for GDExtension.
+	</brief_description>
+	<description>
+		[OpenXRAPIExtension] makes OpenXR available for GDExtension. It provides the OpenXR API to GDExtension through the [method get_instance_proc_addr] method, and the OpenXR instance through [method get_instance].
+		It also provides methods for querying the status of OpenXR initialization, and helper methods for ease of use of the API with GDExtension.
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="can_render">
+			<return type="bool" />
+			<description>
+				Returns [code]true[/code] if OpenXR is initialized for rendering with an XR viewport.
+			</description>
+		</method>
+		<method name="get_error_string">
+			<return type="String" />
+			<param index="0" name="result" type="int" />
+			<description>
+				Returns an error string for a given [code]XrResult[/code].
+			</description>
+		</method>
+		<method name="get_instance">
+			<return type="int" />
+			<description>
+				Returns the [code]XrInstance[/code] craated at the initialization of [code]OpenXRAPI[/code].
+			</description>
+		</method>
+		<method name="get_instance_proc_addr">
+			<return type="int" />
+			<param index="0" name="name" type="String" />
+			<description>
+				Returns the function pointer of the OpenXR function with the specified name, cast to an integer.
+				[code]openxr/util.h[/code] contains macros for ease of use in acquiring OpenXR functions, e.g. [code]GDEXTENSION_INIT_XR_FUNC_V(xrCreateAction)[/code].
+				If the function with the given name does not exist, the method returns [code]0[/code].
```

Comment:
```suggestion
				Returns the function pointer of the OpenXR function with the specified name, cast to an integer. If the function with the given name does not exist, the method returns [code]0[/code].
				[b]Note:[/b] [code]openxr/util.h[/code] contains utility macros for acquiring OpenXR functions, e.g. [code]GDEXTENSION_INIT_XR_FUNC_V(xrCreateAction)[/code].
```

Reply:
@YuriSizov I've updated the documentation accordingly